drivers/flash/flash_stm32_ospi.c

```diff
@@ -41,6 +41,12 @@ LOG_MODULE_REGISTER(flash_stm32_ospi, CONFIG_FLASH_LOG_LEVEL);
 /* used as default value for DTS writeoc */
 #define SPI_NOR_WRITEOC_NONE 0xFF
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+#define OSPI_CLK_SRC RCC_OSPICLKSOURCE_PLL
+#else
+#define OSPI_CLK_SRC RCC_OSPICLKSOURCE_SYSCLK
+#endif /* CONFIG_SOC_SERIES_STM32H7X */
+
```

Comment:
```suggestion
```

Can be removed as it is not used anymore.

Reply:
line removed